src/main/java/io/vertx/core/net/KeyCertOptions.java

```diff
@@ -39,6 +39,17 @@
    */
   KeyManagerFactory getKeyManagerFactory(Vertx vertx) throws Exception;
 
+  /**
+   * Like {@link #getKeyManagerFactory}, except the underlying keystore can be filtered.
+   * <p>
+   * This should be called only when creating a {@link KeyManagerFactory} for a TCP or HTTP server.
+   *
+   * @param vertx the vertx instance
+   * @return the key manager factory
+   * @see JksOptions#setAlias(String)
+   */
+  KeyManagerFactory getKeyManagerFactory(Vertx vertx, boolean filter) throws Exception;
```

Comment:
is this used because the operation should only happen on the server ?

Reply:
Yes